src/ml/neural_net/mxnet_compute_context.cpp

```diff
@@ -0,0 +1,65 @@
+/* Copyright © 2018 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+*/
+
+#include <iostream>
+#include <vector>
+
+#include <external/pybind11/pybind11.h>
```

Comment:
Since we already have `src/external` in the include path ( see https://github.com/apple/turicreate/blob/master/CMakeLists.txt#L1156 ), you can just use `#include <pybind11/pybind11.h>`. Same for the line below.

Reply:
I see. I will change this.